saleor/dashboard/collection/views.py

```diff
@@ -0,0 +1,67 @@
+from django.template.response import TemplateResponse
+from django.contrib import messages
+from django.contrib.auth.decorators import permission_required
+from django.core.urlresolvers import reverse
+from django.http import JsonResponse
+from django.shortcuts import redirect, get_object_or_404
+from django.utils.translation import pgettext_lazy
+
+from ...product.models import Collection
+from ...core.utils import get_paginator_items
+from ..views import staff_member_required
+from .forms import CollectionForm
+
+
+@staff_member_required
+@permission_required('product.view_product')
+def collection_list(request):
+    collections = Collection.objects.prefetch_related('products').all()
+    collections = get_paginator_items(collections, 30, request.GET.get('page'))
+    ctx = {'collections': collections}
+    return TemplateResponse(request,
```

Comment:
We don't use this line-break style, instead there should be break before `request`. The same applies to the `collection_delete` view.